CHANGES.md

```diff
@@ -1389,6 +1389,23 @@ breaking changes, and mappings for the large list of deprecated functions.
 
    *Raja Ashok*
 
+ * Added a new concept for OpenSSL plugability: providers.  This
+   functionality is designed to replace the ENGINE API and ENGINE
+   implementations, and to be much more dynamic, allowing provider
```

Comment:
I'm still unsure about the comma before the `and`.

Reply:
Well, now it actually makes sense, as it shows that "ENGINE API and ENGINE implementations" are a unit.  It makes references clearer